docs/reference-guides/block-api/block-metadata.md

```diff
@@ -421,6 +421,34 @@ It provides structured example data for the block. This data is used to construc
 
 See the [the example documentation](/docs/reference-guides/block-api/block-registration.md#example-optional) for more details.
 
+### Variations
+
+- Type: `array`
+- Optional
+- Localized: Yes (`title`, `description`, and `keywords` of each variation only)
+- Property: `variations`
+
+```json
+{
```

Comment:
Can we also include a simplified snippet in the example at the top of the document in the opening section?

Reply:
Done in https://github.com/WordPress/gutenberg/pull/42539/commits/081f27e8fbcb017bf5c24431b0fdb94f425fbdce